iam-policy-rds.tf

```
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2018 Expedia Inc.
+ * Copyright (C) 2018-2020 Expedia Inc.
```

```suggestion
 * Copyright (C) 2018-2020 Expedia, Inc.
```
etc. for the rest below, seeing as we're updating the year we might as well fix the comma too.